### DISCUSSION TITLE
Welcome to Windows11-React Discussions!

### DISCUSSION THREAD

**Discussion by yichen9247** [yichen9247/Windows11-React#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danishagarwal** [comment]
I am wondering if React is so fast, why are some people still based using jQuery to create applications :? 
What does jQuery offer better than react?
